operations/github/ent-mfa/implementation_plan.md

```diff
@@ -0,0 +1,58 @@
+
+It will impact 1 of the 225 members in OpenSSF enterprise unless the member enables 2FA before the change implementation time. The implementer has reached out the member via Slack and email. 
+
+![one member has 2FA disabled](https://github.com/ossf/staff/blob/f5bb529a91cb013ba6c802cf9b64bc01e5fb2cc9/2FA_not_enabled_query_result.png)
+
+![email](https://github.com/ossf/staff/blob/f5bb529a91cb013ba6c802cf9b64bc01e5fb2cc9/Iimpacted_individual_outreach_email.png)
+
+![slack](https://github.com/ossf/staff/blob/f5bb529a91cb013ba6c802cf9b64bc01e5fb2cc9/Iimpacted_individual_outreach_slack.png)
+
+Worst case scenario, if the member is not enabling 2FA by the implementation time, the implementer will coordinate with the member and the member's organization to restore their access and permissions. 
+An organization owner can reinstate their access privileges and settings if they enable two-factor authentication for their account within three months of their removal from your organization.
+For more information, see "Reinstating a former member of your organization."Members and outside collaborators who will be removed from the organizations owned by your enterprise. The access can be reinstated by setting them  
+
+## Implementation
+Publish Announcement first:
+1. Log into GitHub.
+2. In the top-right corner of GitHub, click your profile photo, then click "Your enterprises".
+3. Click "Settings" next to "Open Source Security Foundation" enterprise.
+4. Under "Settings", click "Announcement".
+5. Fill in all the information as shown in screenshot, then click "Publish Announcement"
+   ![announcement](https://github.com/ossf/staff/blob/f5bb529a91cb013ba6c802cf9b64bc01e5fb2cc9/enterprise_annoucnemen_preview.png)
+7. Verify announcement published by navigating to personal forked staff repo.
+
+Follow [GitHub instructions](https://docs.github.com/en/enterprise-cloud@latest/admin/enforcing-policies/enforcing-policies-for-your-enterprise/enforcing-policies-for-security-settings-in-your-enterprise#requiring-two-factor-authentication-for-organizations-in-your-enterprise-account) for this change:
+   
+1. Under "Settings", click "Authentication security".
+2. Check the box next to  "Require two-factor authentication for all organizations in the Open Source Security Foundation enterprise".
+3. Click "Save".
+4. If prompted, read the information about members and outside collaborators who will be removed from the organizations owned by your enterprise.
+To confirm the change, type your enterprise's name, then click "Remove members & require two-factor authentication".
+Optionally, if any members or outside collaborators are removed from the organizations owned by your enterprise,
+we recommend sending them an invitation to reinstate their former privileges and access to your organization. Each person must enable two-factor authentication before they can accept your invitation.
```

Comment:
Would you go ahead and re-invite the removed member(s) if they still haven't enabled 2FA at this time, so that they have the ability to immediately accept the invite once they've enabled 2FA?

Reply:
If I cannot get hold of the member by the time of the implementation. the org owner needs to do it. It's documented in the "Impacts" section